README.md

```diff
@@ -130,15 +135,16 @@ withdraw_confirm = { gas = 3000000, gas_price = 1000000000 }
 #### transaction options
 
 - `transaction.deposit_relay.gas` - specify how much gas should be consumed by deposit relay
-- `transaction.deposit_relay.gas_price` - specify gas price for deposit relay
 - `transaction.deposit_relay.concurrency` - how many concurrent transactions can be sent (default: **100**)
 - `transaction.withdraw_confirm.gas` - specify how much gas should be consumed by withdraw confirm
-- `transaction.withdraw_confirm.gas_price` - specify gas price for withdraw confirm
 - `transaction.withdraw_confirm.concurrency` - how many concurrent transactions can be sent (default: **100**)
 - `transaction.withdraw_relay.gas` - specify how much gas should be consumed by withdraw relay
-- `transaction.withdraw_relay.gas_price` - specify gas price for withdraw relay
 - `transaction.withdraw_relay.concurrency` - how many concurrent transactions can be sent (default: **100**)
 
+#### transaction options when building with `deploy` feature
```

Comment:
this functionality is not supported any more.
Official bridge contracts are being deployed as per procedure described here: https://github.com/poanetwork/poa-bridge-contracts/tree/master/deploy

Reply:
Ok, good to know, thanks.